src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -0,0 +1,477 @@
+import * as React from 'react';
+import throttle from 'lodash-decorators/throttle';
+import isEqual from 'lodash/isEqual';
+import {autobind} from '@shopify/javascript-utilities/decorators';
+import {
+  addEventListener,
+  removeEventListener,
+} from '@shopify/javascript-utilities/events';
+import {classNames} from '@shopify/react-utilities/styles';
+import {CSS_VAR_PREFIX} from '../../utilities';
+import {Props as RangeSliderProps, DualValue} from '../../types';
+import Labelled from '../../../Labelled';
+import EventListener from '../../../EventListener';
+import {Key} from '../../../../types';
+
+import * as styles from './DualThumb.scss';
+
+export interface State {
+  value: DualValue;
+  trackWidth: number;
+  trackLeft: number;
+  prevValue?: DualValue;
+}
+
+export interface Props extends RangeSliderProps {
+  value: DualValue;
+  id: string;
+  min: number;
+  max: number;
+  step: number;
+}
+
+interface KeyHandlers {
+  [key: string]: () => void;
+}
+
+enum Control {
+  Lower,
+  Upper,
+}
+
+const THUMB_SIZE = 24;
+const OUTPUT_TIP_SIZE = 8;
```

Comment:
We removed the tip from tooltips and popover, do we want it here?

Reply:
That's a good point. I didn't realize we'd also removed it from tooltips, I thought it was just the popover. If we're not using them anymore, then I agree it makes sense to remove it here too.

Reply:
I may actually do this in a followup PR.

Reply:
https://github.com/Shopify/polaris-react/pull/988